src/@types/Verification.d.ts

```diff
@@ -0,0 +1,27 @@
+export interface RegisterVPPayload {
+  signature: string
+  hashedMessage: string
+  fileUrl: string
+}
+
+type RegistryApiResponseBody = SignatureMessageBody | VpDataBody
+
+export interface RegistryApiResponse<RegistryApiResponseBody> {
+  data: {
+    data: RegistryApiResponseBody
+    message: string
+  }
+}
+
+export interface SignatureMessageBody {
+  message: string
+}
+
+export interface VpDataBody {
+  _id: string
+  address: string
+  created: number
+  updated: number
```

Comment:
question(blocking): I think we do not use these properties anymore? Let's remove them if they are not needed.

Reply:
agreed